src/CMakeLists.txt

```diff
@@ -39,3 +40,11 @@ set (src_DIST
    ${src_kms_message_DIST}
    PARENT_SCOPE
 )
+
+
+if (CMAKE_CXX_COMPILER)
```

Comment:
Oh nice! It seems I jumped the gun filing CDRIVER-4159 to add a task to Evergreen. Based on [Evergreen logs](https://evergreen.mongodb.com/lobster/evergreen/task/mongo_c_driver_windows_2015_debug_compile_no_align_patch_43a7061637b6ae783dabe784b18286256e745c49_6141486f3e8e8601b223cd17_21_09_15_01_12_17/0/task#bookmarks=0%2C17144&l=1&shareLine=1617) it looks like this is indeed running on Windows variants.